### ISSUE TITLE
Google service - "This browser or app may not be secure"

### ISSUE THREAD

**Issue by virgiliomsilva** [meetfranz/franz#1720]
**Describe the bug**
When logging in to a Google service it doesn't allow on the basis of being a non-secure app.

**To Reproduce**
1. Add Gmail as Service
2. Insert username and password
3. See error

![image](https://user-images.githubusercontent.com/44083872/70742955-a0a6e680-1d16-11ea-9dbc-d61f2df497e4.png)

**Desktop (please complete the following information):**
 - OS:  Windows 10 Home - 10.0.18363 Build 18363
 - Version 5.3.3

**Comment by MaciejWWojcik** [comment]
I have the same issue!

**Comment by antchien** [comment]
Same issue: MacOS Mojave (version 10.14.6)

**Comment by tkinali** [comment]
Same issue on Linux Mint 19.2 (Franz ver. 5.4.0)

**Comment by MijoBarbaric** [comment]
Same issue here for GMail and Google Calendar.
MacOS Catalina 10.15.1

I tried to re-install Franz from scratch but this didn't help.

**Comment by JeffHowden** [comment]
Same issue here, too, but only for one of the Google accounts I'm trying to use in Franz. Another account has no such issues.

Windows 10
Version: 5.3.3
Release: 10.0.18362

**Comment by j00bar** [comment]
Same issue. Fedora 31, Franz 5.3.3.

**Comment by vanton1** [comment]
same here with Franz 5.4.0-beta on macOS Catalina

**Comment by AleksandarGjorgievski** [comment]
Google recently banned few linux browsers, marking them as not secure. Most likely the underlying browser that Franz is using behind the scenes is the one of them.

**Comment by kacezet** [comment]
Hangouts don't work  -> "This browser or app may not be secure"
MacOS Catalina 10.15.1 & Franz 5.3.3.1582

**Comment by q-g-j** [comment]
Changing the user agent works for me.

1. Close Franz
2. open /home/jann/.config/Franz/recipes/hangouts/index.js in a text editor
3. as the last line add:

`module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`

**Comment by kacezet** [comment]
Yeah @q-g-j it's works! But.. I've other path:
~/Library/ApplicationSupport/Franz/recipes/hangouts/index.js

**Comment by ottomata** [comment]
Same issue, Franz  5.4.1  and MacOS Catalina 10.15.2.

**Comment by dizzu** [comment]
For Windows, the location of the file is: C:\Users\[username]\AppData\Roaming\Franz\recipes\[google_service]\index.js

**Comment by trboltom** [comment]
Same issue for my gmail and google calendar, I have spent over an hour with google support, we have tried everything possible and came to the conclusion that its a problem with the application

**Comment by kayintveen** [comment]
I needed a different url

`~/Library/ApplicationSupport/Franz/recipes/gmail/index.js`
But potentially `gmail` needs to replaced by the google service you need.

My file now looks like this:

```
"use strict";

//module.exports = Franz => Franz;
module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };
```

**Comment by kytwb** [comment]
We are seeing this in Ferdi as well and reckon it would be easier for each user to be able to override the user agent in the service settings as a more sustainable solution:

> As seen in #263, #229 and #140, Google (and maybe other services) are regularly making changes to prevent usage of their services within some browsers/apps. It seems this detection is based on the user agent. If we had remote update of services (still pending via getferdi/server#14), we could update the user agent used in the recipe to periodically overcome this, but ultimately the updated user agent will be somewhat blocked. This is why it would be useful for each user to be able to override the user agent used by a specific service in the service settings, so each user can use whatever user agent works for them (and avoid using the same user agent for all Ferdi user via a recipe update).

Via https://github.com/getferdi/ferdi/issues/264

**Comment by jeroenpeeters** [comment]
Same issue here! Please fix.

**Comment by DoGab** [comment]
Same issue here with Franz 5.4.1 and Ubuntu 19.10.

**Comment by andersonlzcosta** [comment]
Same issue with Franz 5.4.0 and Ubuntu 18.04.3 LTS

**Comment by paradonym** [comment]
> Changing the user agent works for me.
> 
> 1. Close Franz
> 2. open ~/.config/Franz/recipes/hangouts/index.js in a text editor
> 3. as the last line add:
> 
> `module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`

That's an Edge-Browser for which Google always shows warnings to support Google Chrome.
I'd use your usual browsers user agent.

On Windows the file path is %AppData%\Franz\recipes

someone should update the chrome engine in Franz to close this issue.

**Comment by andersonlzcosta** [comment]
> > Changing the user agent works for me.
> > 
> > 1. Close Franz
> > 2. open ~/.config/Franz/recipes/hangouts/index.js in a text editor
> > 3. as the last line add:
> > 
> > `module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`
> 
> That's an Edge-Browser for which Google always shows warnings to support Google Chrome.
> I'd use your usual browsers user agent.
> 
> On Windows the file path is %AppData%\Franz\recipes
> 
> someone should update the chrome engine in Franz to close this issue.

This solved to me!

**Comment by phuzion** [comment]
As of today, this issue seems to have been fixed? Perhaps Google whitelisted the Franz user agent?

I'm running 5.3.3 on Windows 10 Pro.

**Comment by lawrencegs** [comment]
Still problematic on mine. 5.4.1 on Mac Catalina 10.15.1

**Comment by REBELinBLUE** [comment]
Weirdly enough it only seems to be on login. I have the same issue with GMail and Google Calendar, I made this change to the gmail index.js and it let me log in. But then if I clicked any links in emails it said it had been blocked by a popup blocker. I closed franz and removed it, when I reopened I can use gmail as usual. So I guess google is only checking the user agent at log in 🤦‍♂️

**Comment by JeffHowden** [comment]
Yes, that check is only at login. However, if you quit Franz and didn’t have to login, it’s probably because you didn’t fully quit Franz. If you’d fully quit, you'd almost certainly have to login again.

**Comment by REBELinBLUE** [comment]
Huh, what? I fully quit Franz all the time without having to log back into my services

**Comment by trboltom** [comment]
> However, if you quit Franz and didn’t have to login, it’s probably because you didn’t fully quit Franz. If you’d fully quit, you'd almost certainly have to login again.

Not really, you are also not required to login to gmail every single time you quit your browser.

**Comment by crazyhalls** [comment]
> Changing the user agent works for me.
> 
> 1. Close Franz
> 2. open ~/.config/Franz/recipes/hangouts/index.js in a text editor
> 3. as the last line add:
> 
> `module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`

In my case the problem was with the Todoist app, so if you guys are with the same problems with other apps just find the app folder under ~/.config/Franz/recipes and then paste the code above changing the class (normally is the app name in CamelCase).

So i did:
> 1. Close Franz
> 2. open ~/.config/Franz/recipes/todoist/index.js in a text editor
> 3. as the last line add:
> 
> `module.exports = Franz => class Todoist extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`

**Comment by mike-seekwell** [comment]
see this reddit thread on how to fix the Electron app

https://www.reddit.com/r/electronjs/comments/eiy2sf/google_blocking_log_in_from_electron_apps/

`app.userAgentFallback = app.userAgentFallback.replace('Electron/' + process.versions.electron, '');`

**Comment by cschubiner** [comment]
Same issue here, but the `module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`  fix did not work for me

**Comment by beckerben** [comment]
> Same issue here, but the `module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };` fix did not work for me

It only worked for me after I quit franz and restarted it.  Simply disabling the service and enabling it again after making the change did not seem to work but the module.exports line does seem to be a fix at the moment!

**Comment by magne4000** [comment]
Appending `Edge/12.10136`  seems indeed to fix the issue, but most of the URLs in mails now open a popup instead of a new tab

**Comment by janot** [comment]
Provided workaround not working for me on Windows =(

After applying different user agent it loads mobile site version and when I click Sign in, redirects to external browser.

**Comment by johnfewell** [comment]
This is the same issue as #1573. Closing one or the other would be helpful.

**Comment by sagivo** [comment]
I've noticed it only happens in some accounts but not all. I have another Google account and didn't this error message while my private one did.

**Comment by Pablorg99** [comment]
Same problem on Ubuntu 18.04.3 LTS and Franz 5.4.0. The solution above worked for me in the services where I had problems (Google Calendar and Trello). Hope this is fixed soon

**Comment by TylerSustare** [comment]
Same problem on OSX Mojave 10.14.6
I'm getting this from trying to automate "sign in with google" though through firebase and getting this same issue.

**Comment by Leajjes** [comment]
BTW the first fix also works for the google hangouts issue I've encountered in the last week with Franz 5.4.1 on Windows 10.

To recap the steps:
Open [user name]\AppData\Roaming\Franz\recipes\hangouts\index.js in a text editor.

Comment out "//module.exports = Franz => Franz;" line.

Add the following line:
module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };

**Comment by viniciusrd4** [comment]
Same issue with Catalina 10.15.2

**Comment by teamtmedia** [comment]
Same issue here.  Have tried the workaround but it constantly opens into a new browser tab/instance.  gmail service.

Bit sad really as I have decided to engage with the 14 day trial after 2 months of using free.  Adding gmail would be a bit of killer application for me as I could (sadly) drop Thunderbird and move my email completely to the cloud.  Without this I can drop all bar 3 quite easily.  The others are just "nice to haves".

Good luck solving this.  I will try again before the end of my trial.  Not a huge amount of money but something :)

**Comment by storkinsj** [comment]
Me too - different application (IM+). 

Not secure... or maybe blocks ad serving? You decide.

**Comment by zboman** [comment]
After several unsuccessful attempts, I managed to get my Gmail service back working again. I imagine this is the case with any Google service. This is what I did to get it working on MacOS 10.14.6.

1. Open `~/Library/Application\ Support/Franz/recipes/gmail/index.js`
2. Change the line: `module.exports = Franz => Franz;` to be the following:

```
module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };
```

3. Quit the entire Franz app
4. Reopen Franz

And then it just started working for me. Hopefully that might help someone else who is trying to get it to work.

**Comment by NumanIbnMazid** [comment]
same here. Manjaro KDE

**Comment by Gabrock94** [comment]
@zboman solution works for me on Linux Mint. Thanks man!

**Comment by djgerry300** [comment]
@zboman 
Thank you for you solution, but it doesn't work for me.
As others have mentioned, using my Chrome agent causes my browser to open the login page instead of Franz.

Using MacOS 10.15.3 and Franz 5.4.1

**Comment by akhmpr** [comment]
having same issue on macOS Catalina

**Comment by leonards** [comment]
Changing the user-agent works for me. I'm using Manjaro Linux and Franz 5.4.1.

What I did is edit the file ~/.config/Franz/recipes/<google-service-im-using> and change the file index.js as follows:
`
// Comment out the original setting
//module.exports = Franz => Franz;

// Add this setting. It seems only Windows user agent can work
module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:54.0) Gecko/20100101 Firefox/73.0"; } };
`

For some reason **only when the OS is Windows ** will this work for me.

**Comment by dvdrhr** [comment]
Same problem here: Windows 10 Education, freshly downloaded and installed franz. Please fix :/

**Comment by dheerajchand** [comment]
> I've noticed it only happens in some accounts but not all. I have another Google account and didn't get this error message while my private one did.

I am having this exact same error. Two of my Google accounts work fine, and the third does not.

**Comment by jeremymeyers** [comment]
Happens with Google Voice as well

**Comment by am** [comment]
Same issue here. 
OS: MacOS 10.15.3 (19D76)
Franz: 5.3.3 (5.3.3.1582)

Changing the useragent did **not** solve the issue for me.

**Comment by dvdrhr** [comment]
> Same issue here.
> OS: MacOS 10.15.3 (19D76)
> Franz: 5.3.3 (5.3.3.1582)
> 
> Changing the useragent did **not** solve the issue for me.

For me neither.

**Comment by faraidoon123** [comment]
> **Describe the bug**
> When logging in to a Google service it doesn't allow on the basis of being a non-secure app.
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> 
> 1. Add Gmail as Service
> 2. Insert username and password
> 3. See error
> 
> ![image](https://user-images.githubusercontent.com/44083872/70742955-a0a6e680-1d16-11ea-9dbc-d61f2df497e4.png)
> 
> **Desktop (please complete the following information):**
> 
> * OS:  Windows 10 Home - 10.0.18363 Build 18363
> Franz: Version 5.4.0

It worked for me 
just copy the code pasted by kayintveen and Press control+R to open Run and type in Run %Appdata% and enter and then go from there to franz and recipes and in my case i can't login into gmail and get in gmail folder and open index with notepad ++ or just right click and select edit option in menu and it opened with windows own notepad and delete all of them and paste this in there
"use strict";

//module.exports = Franz => Franz;
module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };

in my case it was gmail in your case it might be different if it is then change the class gmail name and put your app which is not working and press control+S to save close all and open Franz and see if its working

**Comment by drivencompassion** [comment]
Someone may be assigned to this issue, to update chrome engine in Franz

**Comment by ehausen** [comment]
https://github.com/meetfranz/franz/issues/1771#issuecomment-606446211

#1771

**Comment by zouloux** [comment]
This on worked for me on MacOS Catalina : 
Edited `~/Library/Application Support/Franz/recipes/gmail` : 
```
"use strict";
module.exports = Franz => class useragent extends Franz
{
    overrideUserAgent ()
    {
        return "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36";
    }
};
```
Links opens correctly, no side effects 👍 

**Important** : I needed to remove my Gmail tab and recreate a new because the old tab was unable to connect.

**Comment by devcline** [comment]
Nothing of those worked for me, any other solutions? win 10

```
module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };

module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.130 Safari/537.36"; } };

module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };
```

**Comment by drivencompassion** [comment]
Got correct values from http://www.browser-info.net updated in place of module.exports = Franz => Franz; and still same error.

**Comment by mowgliee** [comment]
what should i use for the class name for google voice?

**Comment by jilavsky** [comment]
Changing user agent fixed my multiple Gmail account issues, but I am unable to log into Google voice with any user agent I try. **There is something different for Google voice.** 
Same Google mail account works with modified user agent and does NOT work with the same user agent for Google voice.

**Comment by viniciusrd4** [comment]
Solved for with me with Franz update 5.5.0-beta.2.1747

**Comment by sthijs** [comment]
Fixed for me as well with Franz update Version 5.5.0-beta.2 (5.5.0-beta.2.1747), on OSX Catalina 10.15.4.   Thank you! 🙏(The previous mentioned solutions did not work for me.)

**Comment by Waleedov** [comment]
But how do I use the beta version, there is no exe for that!

**Comment by SamSwee** [comment]
> This on worked for me on MacOS Catalina :
> Edited `~/Library/Application Support/Franz/recipes/gmail` :
> 
> ```
> "use strict";
> module.exports = Franz => class useragent extends Franz
> {
>     overrideUserAgent ()
>     {
>         return "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36";
>     }
> };
> ```
> 
> Links opens correctly, no side effects 👍
> 
> **Important** : I needed to remove my Gmail tab and recreate a new because the old tab was unable to connect.

This worked for me on Windows 10, Franz version 5.4.0. I changed the class to useragent.

**Comment by explorovatel** [comment]
> Changing the user agent works for me.
> 
> 1. Close Franz
> 2. open ~/.config/Franz/recipes/hangouts/index.js in a text editor
> 3. as the last line add:
> 
> `module.exports = Franz => class Hangouts extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };`

Hello,

Thank you for sharing. I do not understand how to apply for this IT language. I tried to open Terminal in my Macbook pro and copy + paste which doesn't work. I have the issue with the Gmail and G calendar. Would you be able to help me, please? 

Thanks very much in advance!

**Comment by igorbastosib** [comment]
> > **Describe the bug**
> > When logging in to a Google service it doesn't allow on the basis of being a non-secure app.
> > **To Reproduce**
> > Steps to reproduce the behavior:
> > 
> > 1. Add Gmail as Service
> > 2. Insert username and password
> > 3. See error
> > 
> > ![image](https://user-images.githubusercontent.com/44083872/70742955-a0a6e680-1d16-11ea-9dbc-d61f2df497e4.png)
> > **Desktop (please complete the following information):**
> > 
> > * OS:  Windows 10 Home - 10.0.18363 Build 18363
> >   Franz: Version 5.4.0
> 
> It worked for me
> Press control+R to open Run and type in Run %Appdata% and enter and then go from there to franz and recipes and in my case i can't login into gmail and get in gmail folder and open index with notepad ++ or just right click and select edit option in menu and it opened with windows own notepad and delete all of them and paste this in there
> 
> "use strict";
> 
> //module.exports = Franz => Franz;
> module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };
> 
> in my case it was gmail in your case it might be different if it is then change the class gmail name and put your app which is not working and press control+S to save close all and open Franz and see if its working

Dosen't work for me!
Im using Win10, 64bits, just this account I cant loggin my Gmail on Franz, anothers accounts dosnt appears this issue. 
I dont know what to do!

**Comment by mneko** [comment]
Does anyone know what to change the string to for Google Voice? Because it just deletes `module.exports = Franz => class Google Voice extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36"; } };` when I launch the service.

**Comment by Nailuj1** [comment]
same issue with win10 x64

**Comment by sergiufalcusan** [comment]
6 months since there is this issue with Franz and nobody fixes it. Disappointing to see they add paid features instead.

**Comment by yannguegan** [comment]
Same pb after password change, recipe fix does not work either

**Comment by Nailuj1** [comment]
For me the update process didn't work. So I deinstalled the software and
reinstalled the new one which worked fine.

Yann Guégan <notifications@github.com> schrieb am Sa., 30. Mai 2020, 16:26:

**Comment by cSharpProCompany** [comment]
same for me. i added login using google. and it works fine except that i see this error message when start a debugger in vs code. .... can not continue a development

**Comment by Swift-Jr** [comment]
Upgraded to 5.5 and works for me now

**Comment by paradonym** [comment]
just did a clean installation using the version https://chocolatey.org/packages/franz delivers (should be 5.5.0) and got the problem.
Interesting enough - on the same machine on a Windows 10 on a different SSD the problem doesn't appears.

Used Googlebot User Agents to prevent it...

**Comment by bod2000** [comment]
I used Franz 5.4.0 and updating UserAgent to Chrome/42.. fixed this problem for me in "hangoutschat" recipe.
Here is content:
module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };

Just upgraded to 5.5.0 and it stopped to work with message "This browser does not support security keys."
Recipe source code is the same. Looks like something was changed in Franz itself. I see some changes related to UserAgent in this commit  (https://github.com/meetfranz/franz/commit/d7840ba) and I suspect that workaround that changes UserAgent doesn't work anymore.

Can someone help with question - how can I check what UserAgent is used when try to login to Google in "hangoutschat" recipe?  Doe it use overridden value "Chrome/42.0.2...." or default one?

I have Win 7, Franz 5.5.0

**Comment by project-guru** [comment]
I updated to the latest beta version (5.6.0-beta 1) and I was able to log into Hangouts!

**Comment by bod2000** [comment]
I just tried to upgrade to (5.6.0-beta 1) and it still doesnt work. It doesnt ask me to insert security key and new error message is 

Your sign-in settings don't meet your organization's 2-Step Verification policy.
Contact your admin for more info

**Comment by chauzer** [comment]
> Does anyone know what to change the string to for Google Voice? Because it just deletes `module.exports = Franz => class Google Voice extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/74.0.3729.169 Safari/537.36"; } };` when I launch the service.

This worked for me on OSX

**Comment by armandocin** [comment]
> > **Describe the bug**
> > When logging in to a Google service it doesn't allow on the basis of being a non-secure app.
> > **To Reproduce**
> > Steps to reproduce the behavior:
> > 
> > 1. Add Gmail as Service
> > 2. Insert username and password
> > 3. See error
> > 
> > ![image](https://user-images.githubusercontent.com/44083872/70742955-a0a6e680-1d16-11ea-9dbc-d61f2df497e4.png)
> > **Desktop (please complete the following information):**
> > 
> > * OS:  Windows 10 Home - 10.0.18363 Build 18363
> >   Franz: Version 5.4.0
> 
> It worked for me
> Press control+R to open Run and type in Run %Appdata% and enter and then go from there to franz and recipes and in my case i can't login into gmail and get in gmail folder and open index with notepad ++ or just right click and select edit option in menu and it opened with windows own notepad and delete all of them and paste this in there
> 
> "use strict";
> 
> //module.exports = Franz => Franz;
> module.exports = Franz => class Gmail extends Franz { overrideUserAgent() { return "Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.135 Safari/537.36 Edge/12.10136"; } };
> 
> in my case it was gmail in your case it might be different if it is then change the class gmail name and put your app which is not working and press control+S to save close all and open Franz and see if its working

Works for me!
